src/store/index.js

```diff
@@ -3,29 +3,18 @@ import Vuex from 'vuex'
 import { generateQuiz } from '../services/quizes';
 import { ping } from '../services/ping';
 
-Vue.use(Vuex)
+import { default as Quiz } from '../data/quiz';
+import { default as QuizSetupInfo } from '../data/quiz_setup_info';
+import { default as User } from '../data/user';
+import { default as ResultStats } from '../data/result_stats';
```

Comment:
o ile się orientuję to import bez klamerki bierze default i powinno tak działać
```suggestion
import Quiz from '../data/quiz';
import QuizSetupInfo from '../data/quiz_setup_info';
import User from '../data/user';
import ResultStats from '../data/result_stats';
```

Reply:
To jest akurat bardzo ciekawe - mogę przysiąc że z jakiegoś powodu `import X from '...'` zamiast `import { default as X } from '...' }` powodowało że eslint traktował je jako błąd. Z tego co widzę teraz nie ma tego problemu więc pewnie mi się coś przewidziało.

Wrzucę commit z tymi zmianami.